core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -526,8 +526,10 @@ private[log] class Cleaner(val id: Int,
    * Restore the I/O buffer capacity to its original size
    */
   def restoreBuffers() {
-    if(this.readBuffer.capacity > this.ioBufferSize)
+    if(this.readBuffer.capacity > this.ioBufferSize) {
       this.readBuffer = ByteBuffer.allocate(this.ioBufferSize)
+      info("Restoring cleaner I/O buffers from %s bytes to %s bytes".format(readBuffer.capacity,this.ioBufferSize))
```

Comment:
yeah even though we expect them to grow the same way. It would then be useful to log the growth of them separately, next commit does that